public/index.html

```
@@ -125,122 +101,122 @@ <h1 class="text-4xl sm:text-5xl lg:text-6xl font-extrabold tracking-tighter text
                 </div>
                 
-                <div class="mt-12">
-                    <p class="text-slate-400 mb-4 text-center">Example using JavaScript's <code>fetch</code> API:</p>
-                    <pre class="rounded-2xl text-sm shadow-2xl"><code class="language-javascript">const targetUrl = 'https://api.example.com/data';
-const proxyUrl = `https://blkproxy.vercel.app/api/proxy?url=${encodeURIComponent(targetUrl)}`;
+                <div class="mt-12 rounded-2xl overflow-hidden shadow-2xl border border-slate-300/10">
+                    <pre class="text-sm border-0 m-0"><code>const targetUrl = 'https://api.example.com/data';
+const proxyUrl = `https://blkproxy.iambhvsh.in/api/proxy?url=${encodeURIComponent(targetUrl)}`;
 
 fetch(proxyUrl)
   .then(response => response.json())
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)